app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v3.rb

```diff
@@ -19,5 +25,185 @@ def cluster_name_href(href)
         Ovirt::Cluster.find_by_href(rhevm, href).try(:[], :name)
       end
     end
+
+    # Provisioning
+    def get_host_proxy(host, connection)
+      connection ||= ext_management_system.connect
+      host_proxy = connection.get_resource_by_ems_ref(host.ems_ref)
+      GeneralUpdateMethodNamesDecorator.new(host_proxy)
+    end
+
+    def clone_completed?(args)
+      source = args[:source]
+      phase_context = args[:phase_context]
```

Comment:
@durandom We really do not see a way to do this, before this was part of the provisioning and it was always api specific, but we had only one api. Now we have two.
This part is different between the versions so to keep the same logic we need to deal with this separately.